src/view/popup/popup.tsx

```diff
@@ -12,21 +16,20 @@ import { usePopup } from "./usePopup";
 type Page = "landing" | "recording" | "results";
 
 export const Popup = () => {
+    const SelectedCountriesContext =
+        createContext<SelectedCountriesContextType | null>(null);
+    const selectedCountriesContext = useSelectedCountriesContext();
```

Comment:
I'd probably just deconstruct all the methods and functions you need here: const { function1, function2, value1, .. } = useSelectedCountriesContext()